recipes/hpp-plot/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "hpp-plot" %}
+{% set version = "4.15.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/humanoid-path-planner/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4839f29550fe6a30786c0ce56b069d9df2d1e5de275e5257ad81da9a5f8770ec
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name|lower, max_pin='x.x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - pkg-config
+    - doxygen
+    - {{ cdt('mesa-libgl-devel') }}       # [linux]
+    - {{ cdt('mesa-dri-drivers') }}       # [linux]
+    - {{ cdt('libselinux-devel') }}       # [linux]
```

Comment:
are these CDTs necessary? see the qt-main repo for their equivalent conda packages

Reply:
I see the same cdts in https://github.com/conda-forge/qt-main-feedstock/blob/main/recipe/meta.yaml

Do you mean because qt have cdt I do not need to add them here ? I am a bit confused about the cdt strategy